.github/workflows/lint-eslint-core.yml

```diff
@@ -20,6 +20,15 @@ jobs:
             - uses: actions/setup-node@v2
               with:
                   node-version: 14
+            - name: Get yarn cache directory path
```

Comment:
Changes to `.github/workflows/lint-<plugin>.yml` are generated with `ci/generate-workflows.js` and by running `yarn generate:workflows` in root directory.

I can also generate this myself later if needed.

https://github.com/AriPerkkio/eslint-remote-tester/blob/dde37381a955d51af53e2d16a93249414db0bca6/ci/generate-workflows.js#L24-L50

Reply:
I updated `ci/generate-workflows.js` + generated all the workflows + added some helpful guidelines on how to get `ci/generate-workflows.js` running + added a way to quickly iterate on the output thanks to a `SKIP_TESTS` env variable for `ci/generate-workflows.js` (got the yaml output wrong the first two times)